docs/guides/email/02-basic-email-system.md

```diff
@@ -545,64 +572,86 @@ Shell(192.168.100.6) > tree /etc/dovecot/
+* `auth-sql.conf.ext`. SQL user configuration.
 
 #### Some important configuration file parameters
 
-* `protocols = imap pop3 lmtp submission`. Dovecot supported protocols.
-* `listen = *, ::`. A comma separated list of IPs or hosts where to listen in for connections. "*" listens in all IPv4 interfaces, "::" listens in all IPv6 interfaces.
-* `disable_plaintext_auth = yes`. Whether to turn off the plaintext password for authentication. 
-* `auth_mechanisms = `. The type of authentication mechanism whose values can be multiple and separated by Spaces. Values: plain, login, digest-md5, cram-md5, ntlm, rpa, apop, anonymous, gssapi, otp, skey, gss-spnego.
-* `login_trusted_networks= `. Which clients (MUA) are allowed to use Dovecot. It can be a separate IP address, it can be a network segment, or it can be mixed and separated by spaces. Like this-- `login_trusted_networks = 10.1.1.0/24 192.168.100.2`
-* `mail_location = `. For an empty value, Dovecot attempts to find the mailboxes automatically (looking at ~/Maildir, /var/mail/username, ~/mail, and ~/Mail, in that order). However, auto-detection commonly fails for users whose mail directory hasn’t yet been created, so you should explicitly state the full location here, if possible.
-* `mail_privileged_group = `. This group is enabled temporarily for privileged operations. Currently, this is used only with the INBOX when either its initial creation or dotlocking fails. Typically, this is set to "mail" to access /var/mail.
+* `protocols = imap pop3 lmtp submission`: List of protocols to be used.
+* `listen = *, ::`: A comma separated list of IPs or hosts where to listen in for connections. `*` listens in all IPv4 interfaces, `::` listens in all IPv6 interfaces.
+* `disable_plaintext_auth = yes`: Whether to turn off plaintext authentication. 
+* `auth_mechanisms = `: The type of authentication mechanism to be used. Multiple values can be specified and separated by spaces. Values: plain, login, digest-md5, cram-md5, ntlm, rpa, apop, anonymous, gssapi, otp, skey, gss-spnego.
+* `login_trusted_networks= `: Which IP networks are allowed to use Dovecot. It can be a single IP address, a network segment, or both. As an example: `login_trusted_networks = 10.1.1.0/24 192.168.100.2`
+* `mail_location = `: For an empty value, Dovecot attempts to find the mailboxes automatically (looking at `~/Maildir`, `/var/mail/username`, `~/mail`, and `~/Mail`, in that order). However, auto-detection commonly fails for users whose mail directory hasn’t yet been created, so you should explicitly state the full location here, if possible.
+* `mail_privileged_group = `: This group is enabled temporarily for privileged operations. Currently, this is used only with the INBOX when either its initial creation or dotlocking fails. Typically, this is set to `mail` to access `/var/mail`.
 
 #### Modify multiple files
 
+First, edit the Dovecot configuration:
+
+```bash
+Shell(192.168.100.7) > vim /etc/dovecot/dovecot.conf
```

Comment:
@jimcat8  and @neelchauhan  . Especially @jimcat8 

-  what is the correct IP that should appear here -  192.168.100.6 (original) OR 192.168.100.7 (new) ?

Reply:
192.168.100.6